komga-webui/src/locales/en.json

```diff
@@ -427,22 +427,22 @@
     },
     "filename_chooser": {
       "button_choose": "Choose",
-      "field_destination_filename": "Destination file name",
-      "label_source_filename": "Source File Name",
+      "field_destination_filename": "Destination filename",
+      "label_source_filename": "Source filename",
       "table": {
         "existing_file": "Existing File",
         "order": "Order"
       },
-      "title": "Destination File Name"
+      "title": "Destination Filename"
     },
     "password_change": {
       "button_cancel": "Cancel",
       "button_confirm": "Change password",
       "dialog_title": "Change password",
       "field_new_password": "New password",
-      "field_new_password_error": "New password is required.",
```

Comment:
Should have been "A new password is required."

Reply:
we are talking about the field, with the label "New password", that field being required, we use the form "<field name> is required".

Reply:
Your options are:
"A new password is required" or, "New password required".

Reply:
http://www.butte.edu/departments/cas/tipsheets/grammar/articles.html